src/components/Util/Months.tsx

```diff
@@ -11,4 +14,24 @@ export const Months = [
   {name: 'October', value: 10},
   {name: 'November', value: 11},
   {name: 'December', value: 12}
-];
\ No newline at end of file
+];
+
+
+export function shiftDateToMoment(shiftDate: ShiftDate): Moment {
+  console.log('shiftDate ' + shiftDate.year + '-' + shiftDate.month + '-' + shiftDate.day);
+  return moment([shiftDate.year, shiftDate.month - 1, shiftDate.day]);
+}
+
+// Moment to ShiftDate (as JS Date)
+export function momentToShiftDate(momentDate: Moment): ShiftDate {
+  return {year: momentDate.year(), month: momentDate.month() + 1, day: momentDate.date()};
+}
+
+// Moment to ShiftDate (as string, e.g. 'YYYY-MM-DD')
```

Comment:
This isn't necessarily specific to shift dates - maybe call it `momentToIsoString`?